src/redux/action_types/index.js

```diff
@@ -39,3 +39,8 @@ export const COMMENT_DELETE_FAILURE = "COMMENT_DELETE_FAILURE";
 export const LIKE_ARTICLE = "LIKE_ARTICLE";
 export const DISLIKE_ARTICLE = "DISLIKE_ARTICLE";
 export const LIKE_DISLIKE_ARTICLE_ERROR = "LIKE_DISLIKE_ARTICLE_ERROR";
+// profile action types
```

Comment:
These comments is unnecessary, remove it

Reply:
@micahoriaso Indeed. I have removed the comments.